src/components/CheckBox/CheckBox.style.ts

```diff
@@ -22,76 +22,98 @@ export const checkboxWrapperStyle = () => (theme: Theme): SerializedStyles => cs
     border-radius: 100%;
     transition: all 0.2s;
     content: ' ';
-    width: ${rem(50)};
-    height: ${rem(50)};
+    width: ${rem(48)};
+    height: ${rem(48)};
     position: absolute;
   }
   &:hover:before {
-    background: ${transparentize(0.7, theme.utils.getColor('lightGray', 400))};
+    background: rgba(0, 0, 0, 0.05);
   }
 `;
 
-export const checkboxStyle = ({ intermediate, checked }: Props) => (
-  theme: Theme
-): SerializedStyles => css`
-  background: ${checked
-    ? intermediate
-      ? theme.utils.getColor('darkGray', 400)
-      : theme.utils.getColor('branded1', 300, 'normal')
-    : theme.utils.getColor('lightGray', 300)};
-  border: 0;
-  border-radius: ${rem(2)};
-  width: ${rem(26)};
-  height: ${rem(26)};
-
-  position: absolute;
-  opacity: 0; // hide it
-
-  & + label {
-    position: relative;
-    cursor: pointer;
-    padding: 0;
-  }
+const getBackgroundColor = ({ intermediate, checked, filled, theme }: Props & { theme: Theme }) => {
+  let background = '';
 
-  // Box.
-  & + label:before {
-    content: '';
-    transition: all 0.2s;
-    display: inline-block;
-    vertical-align: text-top;
-    width: ${rem(26)};
-    height: ${rem(26)};
-    background: ${theme.utils.getColor('lightGray', 300)};
-    border-radius: ${rem(2)};
-  }
+  //if filled since outlined has the same properties no matter check or intermediate
+  background = filled
+    ? background
+    : `background: inherit; box-shadow: inset 0px 0px 0px ${rem('2px')} ${theme.utils.getColor(
+        'darkGray',
+        700
+      )};`;
 
-  // Box checked
-  &:checked + label:before {
-    background: ${intermediate
-      ? theme.utils.getColor('darkGray', 400)
-      : theme.utils.getColor('branded1', 400, 'normal')};
+  // if checked and no intermediate
+  if (checked && !intermediate && filled) {
+    background = `background: ${theme.utils.getColor('lightGray', 700)}`;
+  } else {
+    background = !filled ? background : `background: ${theme.utils.getColor('lightGray', 400)}`;
   }
 
-  // Disabled state label.
-  &:disabled + label {
-    cursor: not-allowed;
+  return background;
+};
+
+const getSymbolColor = ({ intermediate, filled, theme }: Props & { theme: Theme }) => {
+  let color = '';
```

Comment:
also here as well there is no need for defining a `let color` where your default value is always defined in the else operator

e.g
```
if (!filled) {
    return theme.utils.getColor('darkGray', 700);
}
return intermediate ? theme.utils.getColor('lightGray', 700) : 'white';
```

Reply:
done